### DISCUSSION TITLE
VSCode pilots, please upgrade to version 1.103.337 (or later)

### DISCUSSION THREAD

**Discussion by davecheney** [community/community#64011] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

VSCode pilots, please upgrade to version 1.103.337 (or later). Older versions of the VSCode extension are no longer supported.

Note: If you're running VSCode 1.80 or earlier, you may need to  first upgrade VSCode to unlock newer versions of the Copilot extension.

**Comment by tomfriwel** [comment]
I was using the old version of copilot because I couldn't use it on a remote connection, until today I found that it was no longer supported and I had to upgrade to the latest version.

Now I can't use it in remote development at all.

**Reply by tomfriwel** [reply]
When I used version `v1.109.386` today, it worked fine.

**Comment by naserykarim922** [comment]
Artificial intelligence management
